demos/4_Deep_Dives/03_htc_2022.ipynb

```diff
@@ -342,15 +343,15 @@
     "\n",
     "$$  \\arg\\min_x \\|Ax-y\\|_2^2 + \\alpha \\mathrm{TV}(x) $$\n",
     "\n",
-    "where total variation (isotropic) is defined as $$\\mathrm{TV}(u) = \\|\\nabla u \\|_{2,1} = \\sum \\sqrt{ (\\partial_{y}u)^{2} + (\\partial_{x}u)^{2} }$$. \n",
+    "where total variation (isotropic) is defined as $$\\mathrm{TV}(x) = \\|\\mathbf{D} x \\|_{2,1} = \\sum \\sqrt{ (\\partial_{y}x)^{2} + (\\partial_{x}x)^{2} }$$ \n",
     "\n",
-    "We implement this regularisation using the CIL optimisation toolkit and make use of the (explicit) PDHG algorithm as an optimisation algorithm because of its flexibility. Call this method **($L^{2}-TV$) Explicit PDHG** .\n",
+    "We implement this regularisation using CIL optimisation toolkit and make use of the (explicit) PDHG algorithm as an optimisation algorithm because of its flexibility.\n",
```

Comment:
Please can we use u instead of x for the image because I find dx(x) confusing when the two x's mean different things!

Reply:
we'd have to change it further down as well in the other maths

Reply:
Please also define what capital X and Y are (the spaces)